stable/harbor/questions.yaml

```diff
@@ -0,0 +1,577 @@
+  show_if: "persistence.imageChartStorage.type=gcs"
+  group: "OCI Artifact Storage Settings"
+- variable: persistence.imageChartStorage.gcs.encodedkey
+  required: true
+  default: ""
+  description: "GCS account encoded key"
+  type: multiline
+  label: GCS Account Encoded Key
+  show_if: "persistence.imageChartStorage.type=gcs"
+  group: "OCI Artifact Storage Settings"
+# s3 storage
+- variable: persistence.imageChartStorage.s3.bucket
+  required: true
+  default: ""
+  description: "S3 bucket name"
+  type: string
+  label: s3 Bucket Name
+  show_if: "persistence.imageChartStorage.type=s3"
+  group: "OCI Artifact Storage Settings"
+- variable: persistence.imageChartStorage.s3.region
+  required: true
+  default: "us-west-1"
+  description: "S3 region"
+  type: string
+  label: s3 Bucket Region
+  show_if: "persistence.imageChartStorage.type=s3"
+  group: "OCI Artifact Storage Settings"
+- variable: persistence.imageChartStorage.s3.accesskey
+  required: true
+  default: ""
+  description: "S3 bucket accesskey"
+  type: string
+  label: s3 AccessKey
+  show_if: "persistence.imageChartStorage.type=s3"
+  group: "OCI Artifact Storage Settings"
+- variable: persistence.imageChartStorage.s3.secretkey
+  required: true
+  default: ""
+  description: "S3 bucket secretkey"
+  type: string
+  label: s3 SecretKey
+  show_if: "persistence.imageChartStorage.type=s3"
+  group: "OCI Artifact Storage Settings"
+- variable: persistence.imageChartStorage.s3.regionendpoint
+  default: ""
+  description: "S3 bucket endpoint"
+  type: string
+  label: s3 Bucket Endpoint
+  show_if: "persistence.imageChartStorage.type=s3"
+  group: "OCI Artifact Storage Settings"
+# oss storage
+- variable: persistence.imageChartStorage.oss.bucket
+  required: true
+  default: ""
+  description: "Aliyun oss bucket name"
+  type: string
+  label: Aliyun oss Bucket Name
+  show_if: "persistence.imageChartStorage.type=oss"
+  group: "OCI Artifact Storage Settings"
+- variable: persistence.imageChartStorage.oss.region
+  required: true
+  default: ""
+  description: "Aliyun oss region name"
+  type: string
+  label: Aliyun oss Bucket Region
+  show_if: "persistence.imageChartStorage.type=oss"
+  group: "OCI Artifact Storage Settings"
+- variable: persistence.imageChartStorage.oss.accesskeyid
+  required: true
+  default: ""
+  description: "Aliyun oss accesskey id"
+  type: string
+  label: Aliyun oss AccessKey ID
+  show_if: "persistence.imageChartStorage.type=oss"
+  group: "OCI Artifact Storage Settings"
+- variable: persistence.imageChartStorage.oss.accesskeysecret
+  required: true
+  default: ""
+  description: "Aliyun oss accesskey secret"
+  type: string
+  label: Aliyun oss AccessKey Secret
+  show_if: "persistence.imageChartStorage.type=oss"
+  group: "OCI Artifact Storage Settings"
+# swift storage
+- variable: persistence.imageChartStorage.swift.authurl
+  required: true
+  default: "https://storage.myprovider.com/v3/auth"
+  type: string
+  label: Swift Auth URL
```

Comment:
Hm, this section lacks "description" (and not just this one). Is it intentional, because it would be the same as label? But I think you've had cases like that before, with description same as label...

Reply:
It was lacking a description in the original harbor questions.yaml file. I added them.